test/e2e/tests/dapp-interactions/contract-interactions.spec.js

```diff
@@ -32,6 +33,11 @@ describe('Deploy contract and call contract methods', function () {
         );
         await unlockWallet(driver);
 
+        await tempToggleSettingRedesignedTransactionConfirmations(driver);
+
+        // Navigate to extension home screen
+        await driver.navigate(PAGES.HOME);
```

Comment:
Adding this step will delay significantly all tests. See vid. Could we instead, add one last step inside the `tempToggleSettingRedesignedTransactionConfirmations` to just close the settings button?

https://github.com/user-attachments/assets/95a37b5f-7ffe-4692-a65a-45a82002996d

Reply:
Added